ports/air-ctl/portfile.cmake

```
@@ -1,10 +1,8 @@
-vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO  inie0722/CTL
```

```suggestion
    REPO  58ca0f229aff093ada66955993cada93e01da18c
```
Here should use commit ID, this will avoid download failed with incorrect hash when this port release a new version.

---

`REPO inie0722/CTL` is correct.
`REF v1.0.0` is also okay ... unless upstream breaks history and moves the tag to another commit.

---

> Here should use commit ID, this will avoid download failed with incorrect hash when this port release a new version.

I think it is better to use tags, I will create new tags in future updates.